inst/pkgdown/templates/navbar-package-version.html

```diff
@@ -0,0 +1,6 @@
+<a href="https://ropensci.org"><img src="https://ropensci.org/img/icon_short_white.svg" id="hexlogo" alt="rOpenSci"></a>
```

Comment:
the exact way we'll add this customization depends on https://github.com/r-lib/pkgdown/pull/1887

Reply:
to be updated depending on https://github.com/r-lib/pkgdown/pull/1891